web-admin/src/routes/-/auth/success/+page.svelte

```diff
@@ -0,0 +1,19 @@
+<script>
```

Comment:
Can we move this page to a location that reflects that it's related to the CLI? E.g. `/-/auth/cli/success`?